apps/test-pid-tuning-api/CMakeLists.txt

```diff
@@ -0,0 +1,19 @@
+add_executable(test-pid-tuning-api.${TARGET}-board.elf)
+target_sources(test-pid-tuning-api.${TARGET}-board.elf PRIVATE src/main.cpp)
+target_link_libraries(test-pid-tuning-api.${TARGET}-board.elf
+        PUBLIC
+        PID
```

Comment:
Does PID have to be public here for compilation to succeed?

Reply:
no it does not

Reply:
Ok prefer having everything private